addons/event_sale/tests/test_event_sale.py

```diff
@@ -31,6 +31,13 @@ def setUp(self):
             'event_id': event.id,
         })
 
+        group_sale_user = self.env.ref('sales_team.group_sale_salesman')
+        self.user_salesperson = self.env['res.users'].create({
```

Comment:
You may use the ``mail_new_test_user`` helper (``from odoo.addons.test_mail.tests.common import mail_new_test_user``). That way some context keys are added to ensure a quick user creation that skips unnecessary stuff for tests (mails, password, ...).